source/tutorials/sink-connector.txt

```diff
@@ -0,0 +1,194 @@
+.. _kafka-tutorial-sink-connector:
+
+=====================================================
+Getting Started with the MongoDB Kafka Sink Connector
+=====================================================
+
+Follow this tutorial to learn how to configure a {+sink-connector+}
+to read data from an Apache Kafka topic and write it to a MongoDB
+collection.
+
+Before you start this tutorial, you must complete the steps in the
+Tutorial Setup (TODO: link).
+
+.. procedure::
+   :style:connected
+
+   .. step:: Configure the Sink Connector
+
+      Create an interactive shell session on the tutorial Docker Container
+      using the following command:
+
+      .. code-block::
+         :copyable: true
+
+         docker run --rm --name tutorialshell --network mongodb-kafka-base_localnet -it robwma/mongokafkatutorial:latest bash
+
+      Create a source configuration file called ``simplesink.json`` with
+      the following command:
+
+      .. code-block::
+         :copyable: true
+
+         nano simplesink.json
+
+      Paste the following configuration information into the file and save
+      your changes:
+
+      .. code-block::
+         :copyable: true
+         :emphasize-lines: 5-7
+
+         { "name": "mongo-sink", "config": {
+           "connector.class": "com.mongodb.kafka.connect.MongoSinkConnector",
+           "topics": "Tutorial2.pets",
+           "connection.uri": "mongodb://mongo1",
+           "key.converter": "org.apache.kafka.connect.storage.StringConverter",
+           "value.converter": "org.apache.kafka.connect.json.JsonConverter",
+           "value.converter.schemas.enable": false,
+           "database": "Tutorial2",
+           "collection": "pets"
+           }
+         }
```

Comment:
issue (non-blocking): This formatting isn't consistent with how we normally format JSON, but it matches the config file format in the other PRs. Is this intentional formatting to save vertical space?

Reply:
Good catch. I will update; I forgot to format these copy-paste-from-the-tutorial JSON blocks completely.